SCClassLibrary/Common/Control/ServerStatus.sc

```diff
@@ -225,9 +225,24 @@ ServerStatusWatcher {
 
 	// final actions needed to finish booting
 	prFinalizeBoot {
-		ServerBoot.run(server);
-		server.sync;
-		server.initTree;
+		// this needs to be forked so that ServerBoot and ServerTree will definitely run before
+		// notified is true.
+		fork({
+			ServerBoot.run(server);
+			server.sync;
+			server.initTree;
+
+			this.notified = true;
+			server.changed(\serverRunning);
+		}, AppClock)
+	}
+
+	prRecoverRemoteLogin { |clientID|
```

Comment:
needs to be added to the list of private methods in the help file

Reply:
what is this name supposed to mean, btw? not immediately apparent to me, sorry

Reply:
hm, maybe really not a good name. 
would you find prHandleRemoteLoginWhenAlreadyRegistered clearer?